bool set_tool_idle
bool run_tool_forward
bool run_tool_in_reverse
bool inhibit_all_run_also_manual
bool activate_unload
bool disactivate_unload
bool activate_lift
bool disactivate_lift

